antsibull/cli/antsibull_build.py

```diff
@@ -44,9 +44,19 @@
 
 
 def _normalize_build_options(args: argparse.Namespace) -> None:
-    args.dest_dir = os.path.expanduser(os.path.expandvars(args.dest_dir))
-    if not os.path.isdir(args.dest_dir):
-        raise InvalidArgumentError(f'{args.dest_dir} must be an existing directory')
+    if not os.path.isdir(args.data_dir):
+        raise InvalidArgumentError(f'{args.data_dir} must be an existing directory')
+
+    if args.command not in (
+            'new-ansible', 'single', 'rebuild-single', 'multiple', 'changelog',
+            'new-acd', 'build-single', 'build-multiple'):
+        return
```

Comment:
Is there a reason to check args.data_dir prior to checking if we're processing one of the build commands?

Reply:
Yes, because all builders have `build_parser` as a parent (which defines `--data-dir`), while only the commands listed here have `build_write_data_parser` as a parent (which adds `--dest-data-dir`).

Reply:
Okay, I'd rather split the checks into two separate functions then.  I vaguely recall trying to map each parser in the `parse_args()` function to one `_normalize_*` function to organize the code.

Reply:
Implemented in 961afa7e10ae378066c9f91751b942703e75611f.